### PR TITLE
pkg/ls:

### PR DESCRIPTION
**Pull request by ChriMarMe**
Add test for package functions and structures

Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

**Comment by ChriMarMe** [comment]
```
 gotest.go:165: Test github.com/u-root/u-root/pkg/ls.TestFileInfo/SimpleFile failed:
        === RUN   TestFileInfo/SimpleFile
            ls_linux_test.go:107: "SimpleFile" failed by lookupGroupName. Want: "root", Got: "root"
            --- FAIL: TestFileInfo/SimpleFile (0.04s)
```

Genau

**Comment by ChriMarMe** [comment]
WARUM???

**Comment by ChriMarMe** [comment]
On hold until https://github.com/u-root/u-root/issues/2257 is solved

File: pkg/ls/ls_linux_test.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	"testing"
+	"time"
+)
+
+const lsregex string = "^([rwxSTstdcb\\-lp?]{10})\\s+(\\d+)?\\s?(\\S+)\\s+(\\S+)\\s+([0-9,]+)?\\s+(\\d+)?(\\D+)?(\\d{1,2}\\D\\d{1,2}\\D\\d{1,2})?(\\D{4})?([\\D|\\d]*)"
+
+func TestFileInfo(t *testing.T) {
+	u, err := user.Current()
+	if err != nil {
+		t.Error(err)
```
Comment (MDr164) [inline pkg/ls/ls_linux_test.go:23]: Errors should be more verbose e.g. `t.Errorf("Failed getting current user: %v", err)`

File: pkg/ls/ls_linux_test.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				t.Errorf("%q failed with %q", tt.name, err)
+			}
+			osfi, err := os.Stat(f.Name())
+			if err != nil {
+				t.Errorf("%q failed with %q", tt.name, err)
+			}
+			fi := FromOSFileInfo(f.Name(), osfi)
+
+			if !strings.Contains(fi.Name, tt.filename) {
+				t.Errorf("%q failed by name. Want: %q, Got: %q", tt.name, tt.filename, strings.TrimPrefix(fi.PrintableName(), tt.name))
```
Comment (MDr164) [inline pkg/ls/ls_linux_test.go:78]: Ryan want to "got" first, followed by "want"

**Review by MDr164 — CHANGES_REQUESTED** [review]

**Review by llogen — APPROVED** [review]
lgtm

**Pull request closed without merging**